pkg/data/egress_lists/egress_lists.go

```diff
@@ -58,6 +64,8 @@ func GetGithubEgressList(platformType cloud.Platform) (*github.RepositoryContent
 		path += cloud.AWSHCP.String()
 	case cloud.AWSClassic:
 		path += cloud.AWSClassic.String()
+	case cloud.AWSHCPZeroEgress:
```

Comment:
Where are we validating the response is a private IP?

Reply:
From my understanding, the network-verifier will check if the required endpoints are accessible from the given cluster vpc/subnet.
I am not sure if it is possible to ensure the network is private.

@rafael-azevedo @fahlmant could you help confirm?

Reply:
The requirements for Zero Egress is that the S3/ECR/EC2 endpoints resolve to a private IP. If we cannot do that within network verifier, that's going to not fully verify the VPC is ready for zero egress.